src/sonic-ctrmgrd/tests/ctrmgr_tools_test.py

```diff
@@ -0,0 +1,290 @@
+                        "container_id": "snmp_docker_id"
+                    },
+                    "lldp": {
+                        "system_state": "down",
+                        "current_owner": "kube",
+                        "container_id": "lldp_docker_id"
+                    }
+                }
+            }
+        },
+        common_test.POST: {
+            common_test.STATE_DB_NO: {
+                common_test.FEATURE_TABLE: {
+                    "snmp": {
+                        "system_state": "up",
+                        "current_owner": "kube",
+                        "container_id": "snmp_docker_id"
+                    },
+                    "lldp": {
+                        "system_state": "down",
+                        "current_owner": "kube",
+                        "container_id": "lldp_docker_id"
+                    }
+                }
+            }
+        },
+        common_test.ACTIONS: {
+            "snmp_docker_id": ["kill"]
+        }
+    },
+    5: {
+        common_test.DESCR: "Throw exception in container get",
+        common_test.ARGS: "ctrmgr_tools tag-all",
+        common_test.RETVAL: -1,
+        common_test.PRE: {
+            common_test.STATE_DB_NO: {
+                common_test.FEATURE_TABLE: {
+                    "snmp": {
+                        "current_owner": "kube",
+                        "container_id": common_test.TRIGGER_GET_THROW
+                    }
+                }
+            }
+        }
+    },
+    6: {
+        common_test.DESCR: "Throw exception in container rm",
+        common_test.ARGS: "ctrmgr_tools tag -f throw_on_rm -n test",
+        common_test.RETVAL: -1,
+        common_test.PRE: {
+            common_test.STATE_DB_NO: {
+                common_test.FEATURE_TABLE: {
+                    common_test.TRIGGER_RM_THROW: {
+                        "current_owner": "kube",
+                        "container_id": "any"
+                    }
+                }
+            }
+        },
+        common_test.ACTIONS: {
+            "any": [
+                {
+                    common_test.IMAGE_TAG: {
+                        "image_name": "test",
+                        "image_tag": "latest",
+                        "force": True
+                    }
+                }
+            ],
+            "throw_on_rm": []
+        }
+    },
+    7: {
+        common_test.DESCR: "Throw exception in container get",
+        common_test.ARGS: "ctrmgr_tools kill-all",
+        common_test.RETVAL: -1,
+        common_test.PRE: {
+            common_test.STATE_DB_NO: {
+                common_test.FEATURE_TABLE: {
+                    "snmp": {
+                        "system_state": "up",
+                        "current_owner": "kube",
+                        "container_id": common_test.TRIGGER_GET_THROW
+                    }
+                }
+            }
+        }
+    }
+}
+
+
+class TestCtrmgrTools(unittest.TestCase):
```

Comment:
Why use `unittest.TestCase`? You're already installing pytest via `tests_require=` of setup.py. Prefer pytest over unittest per the rest of the SONiC codebase.

Reply:
Done